connect/api.go

```diff
@@ -57,3 +58,46 @@ func GetProduct(productQuery Product) (Product, error) {
 	}
 	return remoteProduct, nil
 }
+
+// updateSystemAPI resend the system's hardware info to SCC.
+// https://github.com/SUSE/connect/wiki/SCC-API-%28Implemented%29#update-system
+func updateSystemAPI(distroTarget, namespace string, instanceData []byte) error {
+	payload, err := updatePayload(distroTarget, namespace, instanceData)
+	if err != nil {
+		return err
+	}
+	body := bytes.NewReader(payload)
+	_, err = callHTTP("PUT", "/connect/systems", body, nil, authSystem)
+	return err
+}
+
+// updatePayload returns the JSON body needed for the announce and update calls
+func updatePayload(distroTarget, namespace string, instanceData []byte) ([]byte, error) {
```

Comment:
this should be something like `systemUpdatePayload()` as there are other update APIs like `PUT /connect/systems/products`